### PR TITLE
feat: add interchain security protobuf #NTRN-408

### PR DESCRIPTION
**Pull request by swelf19**
Added interchain security proto types
Updated numeric deserizlizer. Any number may be deserialized from either string or numeric representation when as_str, option_as_str or as_str_vec is used.
The CosmwasmExt macro has been updated. A static field `PATH` has been added to the Query-like type, with the value being the server's path query.

Task definition: Add updateParamsPersimmon to chain manager (https://github.com/neutron-org/neutron-dao/tree/main/contracts/dao/neutron-chain-manager) for the following modules:
- consumer
- dynamicfees
- globalfee

All prs:
https://github.com/neutron-org/neutronjs/pull/10
https://github.com/neutron-org/neutronjsplus/pull/69
https://github.com/neutron-org/neutron-integration-tests/pull/368
https://github.com/neutron-org/neutron-dao/pull/116
https://github.com/neutron-org/neutron-std/pull/8
https://github.com/neutron-org/neutron/pull/765

### CODE REVIEW

File: packages/neutron-std/src/serde/mod.rs [hunk 1]
```diff
@@ -1,17 +1,70 @@
...
+    },
+    Deserialize,
+};
+use std::{
+    fmt::{Display, Formatter},
+    marker::PhantomData,
+    str::FromStr,
+};
+
+struct StringOrNumberVisitor<T> {
```
Comment (pr0n00gler) [inline packages/neutron-std/src/serde/mod.rs:14]: Would be nice to have a description for this structure and what does it do

**Review by pr0n00gler — CHANGES_REQUESTED** [review]

**Review by joldie777 — APPROVED** [review]

**Review by sotnikov-s — DISMISSED** [review]

File: packages/neutron-std/src/serde/mod.rs (hunk 1 above)
Comment (swelf19) [reply packages/neutron-std/src/serde/mod.rs:14]: done

File: packages/neutron-std/src/serde/mod.rs [hunk 2]
```diff
@@ -14,7 +14,19 @@ use std::{
...
+// For example, for the struct: 
+// ```rust
+// struct Foo {
+//     pub bar: i32;
+// }
+// ```
+// Both JSON representations `'{"foo":"12"}'` and `'{"foo":12}'` will give the same result upon deserialization:
+// ```json
+// {
+//     "foo": 12
```
Comment (sotnikov-s) [inline packages/neutron-std/src/serde/mod.rs:27]: the field in JSONs must be `bar`, right?

Comment (sotnikov-s) [reply packages/neutron-std/src/serde/mod.rs:27]: and why is the result of deserialization is another JSON? that's a bit confusing. I think you can just remove the result JSON and just leave the line:
```
// Both JSON representations `'{"foo":"12"}'` and `'{"foo":12}'` will give the same result upon deserialization.
```

**Review by pr0n00gler — APPROVED** [review]

**Pull request merged**